src/pineko/theory.py

```diff
@@ -394,7 +394,9 @@ def fk(self, name, grid_path, tcard, pdf):
         q2_grid = operators["Q2grid"].keys()
         # PineAPPL wants alpha_s = 4*pi*a_s
         # remember that we already accounted for xif in the opcard generation
-        alphas_values = [4.0 * np.pi * astrong.a_s(xir * xir * Q2) for Q2 in q2_grid]
+        alphas_values = [
+            4.0 * np.pi * astrong.a_s(xir * xir * Q2 / xif / xif) for Q2 in q2_grid
```

Comment:
Change:
- `Q2` to `muf2`
- `q2_grid` to `muf2_grid`

Reply:
now it is already done - but as said yesterday, instead I would rather have kept `Q2` and adjust the `q2_grid` definition with the division by `xif` ... this way it would be even more natural I think ...

Reply:
The problem is that, since this works for both scheme B and scheme C and since the q2_grid is really a muf2 grid only in the scheme C case, whatever we may write would be inconsistent (or not clear) with either one or the other scheme. 
As it is now (so as @AleCandido suggested) is coherent with scheme C but not with scheme B (because in scheme B case `operators["Q2grid"].keys()` is really Q2 not muf2. Instead what @felixhekhorn suggests would be perfectly coherent with both but less clear for scheme B because we would have `q2_grid = operators["Q2grid"].keys() / xif /xif` with xif=1.0 (and so making no difference). Personally I prefer the @felixhekhorn suggestion but I believe it is just a matter of taste.

Reply:
Q2 does not exist, it is defined only for DIS. It is always the central factorization scale, so let's start give it the proper name, and avoid confusion.